geometry/create-and-edit-geometries/src/main/resources/create_and_edit_geometries/main.fxml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright 2023 Esri.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+
+<?import com.esri.arcgisruntime.mapping.view.MapView?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.paint.Color?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.image.Image?>
+<StackPane fx:controller="com.esri.samples.create_and_edit_geometries.CreateAndEditGeometriesController"
+           xmlns:fx="http://javafx.com/fxml"
+           stylesheets="@style.css">
+    <MapView fx:id="mapView"/>
+    <VBox StackPane.alignment="TOP_RIGHT" maxWidth="120" maxHeight="110" styleClass="panel-region"
+          alignment="CENTER" spacing="6">
+        <padding>
+            <Insets topRightBottomLeft="10"/>
+        </padding>
+        <background>
+            <Background>
+                <fills>
+                    <BackgroundFill>
+                        <fill>
+                            <Color opacity="0.3"/>
+                        </fill>
+                    </BackgroundFill>
+                </fills>
+            </Background>
+        </background>
+        <GridPane vgap="6" hgap="6" alignment="CENTER">
+            <Label GridPane.rowIndex="0" GridPane.columnIndex="0" text="Create"/>
+        </GridPane>
+        <GridPane vgap="6" hgap="6" alignment="CENTER">
+            <Button fx:id="pointButton" onAction="#handlePointButtonClicked" GridPane.rowIndex="0" GridPane.columnIndex="0" text="Point">
+                <graphic>
+                    <ImageView>
+                        <Image url="@point-32.png"/>
+                    </ImageView>
+                </graphic>
+                <tooltip>
+                    <Tooltip text="Point"/>
+                </tooltip>
+            </Button>
+            <Button fx:id="multipointButton" onAction="#handleMultipointButtonClicked" GridPane.rowIndex="0" GridPane.columnIndex="1" text="Multipoint">
+                <graphic>
+                    <ImageView>
+                        <Image url="@multipoint-32.png"/>
```

Comment:
Let's put the images in a subfolder "images", so it's easier to access the fxml/css file